main.go

```diff
@@ -2,6 +2,10 @@ package main
 
 func Divine()  {
 	
+}
+
+func Add2()  {
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`Add2` is unused (deadcode)